packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -169,9 +169,10 @@ export class CommandBarBase extends React.Component<ICommandBarProps, {}> implem
       onClick: this._onButtonClick(item),
     };
 
-    if (item.iconOnly && itemText !== undefined) {
+    if (item.iconOnly && (itemText !== undefined || (item.tooltipHostProps && item.tooltipHostProps.content))) {
+      const content = itemText ? itemText : item.tooltipHostProps!.content;
```

Comment:
We shouldn't actually need this since the spread below will take care of this logic for us.